docs/core/logging.md

```diff
@@ -178,148 +224,286 @@ Powertools for AWS Lambda (Java) extends the functionality of Log4J. Below is an
+        public APIGatewayProxyResponseEvent handleRequest(final APIGatewayProxyRequestEvent input, final Context context) {
+            // ...
+        }
+    }
     ```
 
-- Utility also provides sample template for [Elastic Common Schema(ECS)](https://www.elastic.co/guide/en/ecs/current/ecs-reference.html) layout.
-The field emitted in logs will follow specs from [ECS](https://www.elastic.co/guide/en/ecs/current/ecs-reference.html) together with field captured by utility as mentioned [above](#standard-structured-keys).
+### Logging handler uncaught exception
+By default, AWS Lambda logs any uncaught exception that might happen in the handler. However, this log is not structured
+and does not contain any additional context. You can instruct the `@Logging` annotation to log this kind of exception 
+with `logError` param or via `POWERTOOLS_LOGGER_LOG_ERROR` env var.
 
-    Use `LambdaEcsLayout.json` as `eventTemplateUri` when configuring `JsonTemplateLayout`.
+???+ warning
+    This is disabled by default to prevent double logging
 
-=== "log4j2.xml"
+=== "AppLogResponse.java"
 
-    ```xml hl_lines="5"
-    <?xml version="1.0" encoding="UTF-8"?>
-    <Configuration>
-        <Appenders>
-            <Console name="JsonAppender" target="SYSTEM_OUT">
-                <JsonTemplateLayout eventTemplateUri="classpath:LambdaEcsLayout.json" />
-            </Console>
-        </Appenders>
-        <Loggers>
-            <Logger name="JsonLogger" level="INFO" additivity="false">
-                <AppenderRef ref="JsonAppender"/>
-            </Logger>
-            <Root level="info">
-                <AppenderRef ref="JsonAppender"/>
-            </Root>
-        </Loggers>
-    </Configuration>
+    ```java hl_lines="5"
+    public class AppLogError implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
+    
+        private static final Logger LOGGER = LoggerFactory.getLogger(AppLogError.class);
+        
+        @Logging(logError = true)
+        public APIGatewayProxyResponseEvent handleRequest(final APIGatewayProxyRequestEvent input, final Context context) {
+            // ...
+        }
+    }
     ```
 
-## Setting a Correlation ID
+### Logging additional keys
 
-You can set a Correlation ID using `correlationIdPath` attribute by passing a [JSON Pointer expression](https://datatracker.ietf.org/doc/html/draft-ietf-appsawg-json-pointer-03){target="_blank"}.
+#### Logging a correlation ID
 
-=== "App.java"
+You can set a correlation ID using the `correlationIdPath` attribute of the `@Logging`annotation,
+by passing a [JMESPath expression](https://jmespath.org/tutorial.html){target="_blank"}, 
+including our custom [JMESPath Functions](../utilities/serialization.md#built-in-functions).
 
-    ```java hl_lines="8"
-    /**
-     * Handler for requests to Lambda function.
-     */
-    public class App implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
+=== "AppCorrelationId.java"
```

Comment:
Might want to rename the second code blocks just below with the same names; as it stands, when you click through the tabs here, it moves both the active tab block you click on, plus the other tabs with the same names

Reply:
even with a different name, the tab changes, I don't know why...

Reply:
Fixes it for me (ignore stupid names)

<img width="975" alt="Screenshot 2023-12-08 at 15 17 05" src="https://github.com/aws-powertools/powertools-lambda-java/assets/474021/dbd75d26-921d-46d4-8c24-bb7b128e47be">